packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -21,6 +21,7 @@ const {
     getConfiguredExtensions,
     validateExtensionDependencies
 } = require('@salesforce/pwa-kit-extension-sdk/shared/utils')
+const listOverridesModule = require('@salesforce/pwa-kit-extension-sdk/bin/list-overrides')
```

Comment:
There is no need to import this as a module, it should be in your `/node_modules/.bin` folder and you can execute it like a script from there.